denis/spotify/src/App.tsx

```diff
@@ -0,0 +1,105 @@
+import '@material/image-list/dist/mdc.image-list.css';
+import '@material/switch/dist/mdc.switch.css';
+import '@material/textfield/dist/mdc.textfield.css';
+import { CircularProgress } from '@rmwc/circular-progress';
+import '@rmwc/circular-progress/circular-progress.css';
+import { ImageList, ImageListImage, ImageListImageAspectContainer, ImageListItem, ImageListLabel, ImageListSupporting, } from '@rmwc/image-list';
+import { Switch } from '@rmwc/switch';
+import { TextField, TextFieldHelperText } from '@rmwc/textfield';
+import gql from 'graphql-tag';
+import * as React from 'react';
+import { Query } from 'react-apollo';
+import './App.css';
+
+import { ApolloProvider } from "react-apollo";
+import {Simulate} from "react-dom/test-utils";
+import load = Simulate.load;
+
+const GET_ARTIST = gql`
+    query getArtists($artistName: String){
+        queryArtists(byName: $artistName) {
+            id
+            name
+            image
+        }
+    }
+`;
+
+interface IArtist{
+    id: string;
+    name: string;
+    image: string;
+}
+
+const pictureAlbumCover = (artistName: string, standardLayout: boolean) => (
+    <Query variables={{artistName, standardLayout}} query={GET_ARTIST}>
+        {({loading, error, data}) => {
+            console.log(artistName);
+            if (artistName === ""){ return ""};
```

Comment:
Toto by ti mal picovat aj nejaky `linter`, ale malo by to vyzerat skor takto:
```
if (artistName === "") {
    return "";
}
```
alebo aj
```
if (!artistName) { return ""; }
```

Za blokom nemusi byt `;`
Za returnom byt mala byt `;`
Za `(..)` by mala byt medzera.